cni/network/network.go

```diff
@@ -1073,7 +1076,15 @@ func (plugin *NetPlugin) Delete(args *cniSkel.CmdArgs) error {
 					zap.String("containerID", args.ContainerID))
 				sendEvent(plugin, fmt.Sprintf("Release ip by ContainerID (endpoint not found):%v", args.ContainerID))
 				if err = plugin.ipamInvoker.Delete(nil, nwCfg, args, nwInfo.Options); err != nil {
-					return plugin.RetriableError(fmt.Errorf("failed to release address(no endpoint): %w", err))
+					if errors.As(err, &connectionErr) {
```

Comment:
should this be 
```suggestion
					if errors.Is(err, &cnscli.ConnectionFailureErr{}) {
```
the pointer semantics of errors.As/Is are confusing to me. but it seems wrong to compare `err` to this local `connectionErr *cnscli.ConnectionFailureErr` variable

Reply:
from what I previously gathered in search, errors.As works because it compares the actual err to type ConnectionFailureErr, and it will return true when a match is found in the error chain.